Project/Fall2020_CSC403_Project/FrmBattle.cs

```diff
@@ -63,14 +68,33 @@ public partial class FrmBattle : Form {
       lblEnemyHealthFull.Text = enemy.Health.ToString();
     }
 
-    private void btnAttack_Click(object sender, EventArgs e) {
+    private void btnAttack_Click(object sender, EventArgs e)
+    {
       player.OnAttack(-4);
-      if (enemy.Health > 0) {
+      //SoundPlayer attackSound = new SoundPlayer(Resources.attack_sound1);
+      //attackSound.Play();
+      if (enemy.Health > 0)
+      {
         enemy.OnAttack(-2);
       }
 
       UpdateHealthBars();
-      if (player.Health <= 0 || enemy.Health <= 0) {
+      if (player.Health <= 0)
+      {
+        SoundPlayer deathSound = new SoundPlayer(Resources.death_music);
+        battleSound.Stop();
+        deathSound.Play();
```

Comment:
Would you mind fetching the changes from main? If the player tries to flee and fails, they may also die. Might want to implement the death sound in that case too